svg-aam/role/roles.html

```diff
@@ -0,0 +1,107 @@
+<!doctype html>
+<html>
+<head>
+  <title>SVG-AAM Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+
+<p>Tests the mappings defined in <a href="https://w3c.github.io/svg-aam/#mapping_role_table">SVG-AAM: 6.2 Element Mapping</a>.<p>
+
+<h2>Simple Elements With aria-label to Ensure Tree Inclusion</h2>
+<svg>
+  <!-- Some elements skipped: never-rendered elements can return unpredicable/undefined/unspecified values for computedrole. -->
+  <a href="#" data-testname="el-a[href]" data-expectedrole="link" aria-label="label" class="ex">x</a>
+  <a xlink:href="#" data-testname="el-a[xlink:href]" data-expectedrole="link" aria-label="label" class="ex">x</a>
```

Comment:
As an FYI, `xlink:href` is deprecated in SVG 2: https://svgwg.org/svg2-draft/linking.html#XLinkRefAttrs

Reply:
Seems like we need a new issue on SVG-AAM. Since both WebKit and Chromium pass, I'd like to keep them until Gecko state they don't intend to fix the test, or any engine removes support.

Reply:
https://github.com/w3c/svg-aam/issues/34